packages/runner/lib/app.ts

```diff
@@ -2,6 +2,9 @@ import './tracer.js';
 import { server } from './server.js';
 import { stringifyError } from '@nangohq/utils';
 import { logger } from './utils.js';
+import { monitorProviders } from '@nangohq/shared';
+
+await monitorProviders();
```

Comment:
let's cleanup the `intervalTimer` in the `close` function below

Reply:
Fixed in e70ed3f7